cli/src/pcluster/models/cluster.py

```diff
@@ -326,14 +328,17 @@ def _upload_config(self):
                     config_name=self._s3_artifacts_dict.get("config_name"),
                 )
 
-                # config version will be stored in DB by the cookbook at the first update
+                # config version will be stored in DB by the cookbook
```

Comment:
dynamo db should only be used to communicate to the Lambda that is waiting for the update that the update is successful. Ideally we should not be saving more than one version in Dynamo. What are we currently storing?

Reply:
Without my [PR](https://github.com/aws/aws-parallelcluster-cookbook/pull/948), we only store one version, the version of the config file with implied values. With the PRs, there will be two versions (original config, config with implied values). I used this approach because the code was completed before our discussion. 
Overall, are you suggesting we store the version of the original config file on Cloudformation, and the version of the config with implied values on dynamodb?

Reply:
Restored the previous behavior (Only store the version of the config file with implied values in the dynamoda). And store the version of the original config file in Parameters of CloudFormation